packages/elements/src/space/index.js

```diff
@@ -0,0 +1,39 @@
+const SPACING_BASE = 8;
+
+/**
+ * Converts pixels to rem.
+ *
+ * @param {Number} [px=0]
+ */
+const pxToRem = px => px / 16;
```

Comment:
- This assumes a base font size of 16 -> Would this be a problem if the font size on the html document is set to something different? 

The parameter is documented as optional with a default of `0`, but it doesn't have a default - it'll be undefined
This function is private (not exported) it doesn't need to be documented